packit/cli/init.py

```diff
@@ -67,6 +67,9 @@ def init(
         ),
     }
 
+    with open(working_dir / ".gitignore", mode="a") as gitignore:
+        print("prepare_sources_result*/", file=gitignore)
```

Comment:
For CLI, we use `click.echo` call instead of  `print`... Here, it does not give us much benefits, but I would try to be consistent.

E.g. here:
https://github.com/packit/packit/blob/7a8a09dd4235bd2c8ccac6b6ad9533bd841238c2/packit/cli/create_update.py#L125

Reply:
It's appending to the file, not stdout :eyes:

Reply:
Oh, it supports the files too /o Do I even want to know why :despair:

Reply:
Looks like the only benefit would be immediate flushing

https://github.com/pallets/click/blob/4a758f5c38636ecd9d030356d85bcca3d8fbd0ca/src/click/utils.py#L321-L322

It's being added to the `.gitignore`, so it doesn't really make sense to use any styling for the output, IMO it just adds more complexity on top of a simple operation.